### ISSUE TITLE
VFE encoder has wrong structure.

### ISSUE THREAD

**Issue by Super-Tree** [qianguih/voxelnet#14]
Line 28:[ ` aggregated = tf.reduce_max(pointwise, axis=1, keep_dims=True)`](https://github.com/qianguih/voxelnet/blob/master/model/group_pointcloud.py)
This op calculates all the points' featrue in a voxel including those fake zeros points.That isn't the same with the vfe in the origin paper.Although,the next step has a mask multiplying op，but it can't help your vfe encoder to calculate the right reduce max feature.
If you use qq or wechat, we may discuss use Chinses ^_^

**Comment by burui11087** [comment]
@Super-Tree 
Hi, I want to talk with you by QQ, you can add me by my mail prefix. I also have same implementation issue about vfe

thanks

**Comment by Super-Tree** [comment]
@burui11087  I have already sent an email to you,please check out.
